test/qdmi/test_driver.cpp

```diff
@@ -503,8 +504,9 @@ TEST_P(DriverTest, QueryNeedsCalibration) {
 #ifdef _WIN32
 const std::array<std::string, 1> DEVICES{"MQT NA Default QDMI Device"};
 #else
-const std::array<std::string, 2> DEVICES{"MQT NA Default QDMI Device",
-                                         "MQT NA Dynamic QDMI Device"};
+const std::array<std::string, 3> DEVICES{"MQT NA Default QDMI Device",
+                                         "MQT NA Dynamic QDMI Device",
+                                         "MQT Core DDSIM QDMI Device"};
```

Comment:
I would assume that the DDSIM device should also be added to the windows tests above.

Reply:
Done in https://github.com/munich-quantum-toolkit/core/pull/1336/commits/a1047a8bbd6d73f507b8e88cc88c7ebee25ebf15